app/javascript/react/components/GameTile.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Enzyme, { mount } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+import GameTile from './GameTile'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe("GameTile", () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(
+      <GameTile
+        name="Game 1"
+        description="Nothing matters"
+        playerNum="2-8 who cares"
```

Comment:
we can keep test data simple, but let's try to be mindful of the phrasing so that it is professional and relevant